boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -95,6 +95,15 @@
 		};
 	};
 
+	nrf_radio_coex: nrf7002-coex {
+		status = "okay";
+		compatible = "nordic,nrf700x-coex";
+		req-gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
```

Comment:
Compared to the removed code, `req-gpios` and `status0-gpios` pin numbers were switched. Is that intentional?